docs/sources/mimir/release-notes/v2.11.md

```diff
@@ -0,0 +1,92 @@
+
+- **Block specified queries on a per-tenant basis.** This is configured via the `blocked_queries` limit.
+- **Store metadata when ingesting metrics via OTLP.** This makes metric description and type available when ingesting metrics via OTLP. You can enable this feature by setting the CLI flag `-distributor.enable-otlp-metadata-storage` to `true`.
+- **Reject gRPC push requests that the ingester/distributor is unable to accept before reading them into memory.** You can enable this feature by using the `-ingester.limit-inflight-requests-using-grpc-method-limiter` and/or the `-distributor.limit-inflight-requests-using-grpc-method-limiter` CLI flags for the ingester and/or the distributor, respectively.
+- **Customize the memcached client write and read buffer size.** The buffer allocated for each memcached connection can be configured via the following CLI flags:
+  - For the blocks storage:
+    - `-blocks-storage.bucket-store.chunks-cache.memcached.read-buffer-size-bytes`
+    - `-blocks-storage.bucket-store.chunks-cache.memcached.write-buffer-size-bytes`
+    - `-blocks-storage.bucket-store.index-cache.memcached.read-buffer-size-bytes`
+    - `-blocks-storage.bucket-store.index-cache.memcached.write-buffer-size-bytes`
+    - `-blocks-storage.bucket-store.metadata-cache.memcached.read-buffer-size-bytes`
+    - `-blocks-storage.bucket-store.metadata-cache.memcached.write-buffer-size-bytes`
+  - For the query frontend:
+    - `-query-frontend.results-cache.memcached.read-buffer-size-bytes`
+    - `-query-frontend.results-cache.memcached.write-buffer-size-bytes`
+  - For the ruler storage:
+    - `-ruler-storage.cache.memcached.read-buffer-size-bytes`
+    - `-ruler-storage.cache.memcached.write-buffer-size-bytes`
+- **Configure the number of long-living workers used to process gRPC requests.** This can decrease CPU usage by reducing the number of stack allocations. Configure this feature by using the `-server.grpc.num-workers` CLI flag.
+- **Enforce a limit in bytes on the `PostingsForMatchers()` cache used by ingesters.** This limit can be configured via the `-blocks-storage.tsdb.head-postings-for-matchers-cache-max-bytes` and `-blocks-storage.tsdb.block-postings-for-matchers-cache-max-bytes` CLI flags.
+- ** Pre-allocate the pool of workers in the distributor that are used to send push requests to ingesters.** This can decrease CPU usage by reducing the number of stack allocations. You can enable this feature by using the `-distributor.reusable-ingester-push-worker` flag.
+
+### Helm chart improvements
+
+The Grafana Mimir and Grafana Enterprise Metrics Helm chart is now released independently. See the [Grafana Mimir Helm chart documentation](/docs/helm-charts/mimir-distributed/latest/).
+
+## Important changes
+
+In Grafana Mimir 2.11 the following behavior has changed:
+
+- The distributor `Push()` endpoint will now return the following gRPC codes instead of HTTP status codes:
```

Comment:
As a user, I don't know what is the distributor's `Push()` endpoint, is that where I send my HTTP requests?

Reply:
Hum, I think it's only misleading here. Distributor doesn't have a documented gRPC API as far as I can see. This change is only internal, as this endpoint is called from the http wrapper, and these errors are translated to same as previously.

I would just remove this section.

cc @duricanikolic for more context.